ios/Podfile.lock

```diff
@@ -4708,7 +4711,7 @@ SPEC CHECKSUMS:
   Analytics: 3106b5af9b799c934f98a68ab8ba199f1c603fdb
   AppAuth: 8fca6b5563a5baef2c04bee27538025e4ceb2add
   ASN1Decoder: 6110fdeacfdb41559b1481457a1645be716610aa
-  boost: 57d2868c099736d80fcd648bf211b4431e51a558
+  boost: a7c83b31436843459a1961bfd74b96033dc77234
```

Comment:
please try to `rm -rf ios/Pods ios/build` and do pod install again. stale files in **ios/Pods** used to introduce change like this.